README.md

```diff
@@ -8,12 +8,63 @@
 
 ## Overview
 
-This lab is meant to give students a chance to practice pulling together
-_everything_ that they've learned in this unit.
+In this lab, we're going to build our very own application - a dashboard which shows us tweets from Twitter
+based on a particular keyword in real time. The real twitter API requires authentication which is quite complex
+and out of the scope of this course (feel free to read more about it [here](https://dev.twitter.com/oauth)).
+Thus, we will be using fake data from a mock API - that is, we will hardcode some past tweets an pretend
+the data is coming from the real Twitter API. Not to worry, this is a common practice in software engineering,
+especially at the early stages of a project or for testing.
 
-We can have them build a small but fairly complex (though not complicated)
-application. It'd be great to have them fetch some remote data, selectively
-update state, etc.
+<p align="center">
+  <img src="https://www.rivaliq.com/wp-content/uploads/2015/06/marketing-works-net.png"/>
+</p>
+
+### See some tweets!
+The `<App />` component is our root component and is responsible for fetching the data and passing it through
+to the Tweet wall and our charting library. As you can see, there is a function `fetchTweets()` which handles
+fetching the new lot of tweets and updating the state. Only it's never called! Lets fix that. Use the
```

Comment:
I'd try to be less verbose here since you are more or less instructing them what line of code to write. Maybe something like 'this method should be called when the component will be mounted' or something, to make sure that they still use the right lifecycle hook?